pkg/agent/testdata/AKSUbuntu1604+Containerd/line70.sh

```diff
@@ -312,6 +312,10 @@ ensureContainerd() {
 ExecStartPost=/sbin/iptables -P FORWARD ACCEPT
 EOF
 
+  if [ "${ARTIFACT_STREAMING_ENABLED}" == "true" ]; then
+    ensureArtifactStreaming || $ERR_ARTIFACT_STREAMING_DOWNLOAD_INSTALL
```

Comment:
```suggestion
    ensureArtifactStreaming || $ERR_ARTIFACT_STREAMING_INSTALL
```

Reply:
https://github.com/Azure/AgentBaker/blob/2703271668c194b0776e9ed80e15b4230e29b3dd/parts/linux/cloud-init/artifacts/cse_helpers.sh#L80C32-L80C32

will need to be changed at origin and then here.

Reply:
Split it into two error codes